app/src/main/java/chat/rocket/android/wallet/presentation/WalletView.kt

```diff
@@ -0,0 +1,3 @@
+package chat.rocket.android.wallet.presentation
+
+interface WalletView
```

Comment:
Does this do interface do anything for us?

Reply:
It may need to be fleshed out more... it should define UI related functions that the WalletFragment implements. I believe a couple of functions currently in WalletFragment should be in this interface/view, which I can add now